libqtile/widget/mpd2widget.py

```diff
@@ -257,6 +257,8 @@ def formatter(self, status, current_song):
         song_info = defaultdict(lambda: default)
         song_info["play_status"] = self.play_states[status["state"]]
 
+        status["currentsong"] = str(int(status["song"])+1)
```

Comment:
I don't think you need the `str()` conversion as this is just used for string formatting so an integer is fine.

Is `song` always available and always an integer? If not, this line is going to fail.

You'll also need to update the `test_mpd2widget.py` file as the field is not available there.

Reply:
Can you also add a test that shows this field being used.